pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "nwp-demo"
+# Keep only Major and Minor version in this configuration,
+# the patch version is calculated when releasing the library
+version = "1.0"
+description = "Demo scripts and notebooks for reading and processing forecast data using Polytope."
+authors = ["MeteoSwiss"]
+package-mode = false
+
+[[tool.poetry.source]]
+name = "pypi-mch"
+url = "https://service.meteoswiss.ch/nexus/repository/python-all/simple"
+
+[tool.poetry.dependencies]
+python = "~3.11"
+meteodata-lab = { git = "https://github.com/MeteoSwiss/meteodata-lab.git", branch = "main", extras = ["all"] }
+pyfdb = "^0.0.3"
+polytope-client = "^0.7.4"
+matplotlib = "^3.8.4"
+cartopy = "^0.22"
+eccodes = ">=1.7.1"
```

Comment:
The version of eccodes needs to match the C library version for 2.37+, before that it needs to be pinned to the last 1.x version